packages/@uppy/aws-s3-multipart/src/index.js

```diff
@@ -436,6 +438,11 @@ export default class AwsS3Multipart extends BasePlugin {
     return Promise.all(promises)
   }
 
+  setCompanionHeaders () {
```

Comment:
Let's not add public methods if it's not necessary.
```suggestion
  #setCompanionHeaders = () => {
```

Reply:
Thanks, I've updated the code.